From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: James Shkolnik <jshkolnik@gmail.com>
Date: Fri, 12 Jun 2026 00:00:00 +0000
Subject: orangepi4pro: set human-readable model string

The shared sun60iw2p1.dtsi sets model = "sun60iw2"; override it in the board
dts so /proc/device-tree/model and Armbian report "Orange Pi 4 Pro". Cosmetic.
---
 arch/arm64/boot/dts/allwinner/sun60i-a733-orangepi-4-pro.dts | 1 +
 1 file changed, 1 insertion(+)

diff --git a/arch/arm64/boot/dts/allwinner/sun60i-a733-orangepi-4-pro.dts b/arch/arm64/boot/dts/allwinner/sun60i-a733-orangepi-4-pro.dts
index 111111111111..222222222222 100644
--- a/arch/arm64/boot/dts/allwinner/sun60i-a733-orangepi-4-pro.dts
+++ b/arch/arm64/boot/dts/allwinner/sun60i-a733-orangepi-4-pro.dts
@@ -6,6 +6,7 @@
 //#define TYPEC_DP 1
 /{
 	board = "A733";
+	model = "Orange Pi 4 Pro";
 	compatible = "xunlong,orangepi-4-pro", "arm,sun60iw2p1";
 	aliases {
 		arisc-config = &arisc_config;
-- 
Armbian

